completion_aggregator/models.py

```diff
@@ -145,6 +145,7 @@ def submit_completion(self, user, course_key, block_key, aggregation_name, earne
                 'percent': percent,
                 'possible': possible,
                 'earned': earned,
+                'last_modified': last_modified,
```

Comment:
I added this field (even though there's an auto-updated "modified" field, because the way that aggregators are updated from block completions means that relying on the actual time it was updated introduces race conditions that could cause completions to get missed.  `last_modified` is updated to the timestamp of the most recently seen block completion, so any newer objects will have a newer timestamp & there's no race condition.

Reply:
It's probably worth mentioning this in the docstring for this method, or maybe the model's docstring.